openff/nagl_models/_dynamic_fetch.py

```diff
@@ -91,7 +92,11 @@ def get_model(
     # See if it's available in the openff-nagl-models python package
     try:
         file_path = validate_nagl_model_path(filename)
-        assert_hash_equal(file_path, file_hash)
+        # If filename happens to be an absolute path (not guaranteed this is in scope, but is temporarily supported,
+        # see https://github.com/openforcefield/openff-nagl-models/issues/68) the hash check will be skipped.
+        # This isn't a final decision on any behaviors, just a temporary workaround.
+        if file_hash is not None:
+            assert_hash_equal(file_path, file_hash)
```

Comment:
Any reason why trimming the full path down to the filename wouldn't work?

Reply:
I figured that, if someone has a file whose name corresponds to a known hash, but for some reason they've put it in a different place, it's not clear that they intend for the hash to be compared. My goal was to NOT specify any particular behavior for this undecided case, and so NOT checking the hash made a bit more sense.